zio-cli/shared/src/main/scala/zio/cli/CliApp.scala

```diff
@@ -115,8 +185,9 @@ object CliApp {
           case Command.Subcommands(parent, _) => prefix(parent)
         }
 
+      val arg_from_config_files = loadOptionsFromFile(self.name)
       self.command
-        .parse(prefix(self.command) ++ args, self.config)
+        .parse(prefix(self.command) ++ arg_from_config_files ++ args, self.config)
```

Comment:
Hi! I don't think including `arg_from_config_files` there would work. It's probably best to do this inside `command.parse` so you can have more control in how args are parsed.

Reply:
ok move it inside `command.parse`